src/test/regress/expected/multi_metadata_access.out

```diff
@@ -0,0 +1,89 @@
+--
+-- MULTI_METADATA_ACCESS
+--
+ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 1360000;
+ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 1360000;
+CREATE USER no_access;
+NOTICE:  not propagating CREATE ROLE/USER commands to worker nodes
+HINT:  Connect to worker nodes directly to manually create all necessary users and roles.
+SET ROLE no_access;
+SELECT COUNT(*) FROM pg_dist_partition;
+ count 
+-------
+     0
+(1 row)
+
+SELECT COUNT(*) FROM pg_dist_shard;
+ count 
+-------
+     0
+(1 row)
+
+SELECT COUNT(*) FROM pg_dist_shard_placement;
+ count 
+-------
+     0
+(1 row)
+
+SELECT COUNT(*) FROM pg_dist_local_group;
+ count 
+-------
+     1
+(1 row)
+
+SELECT COUNT(*) FROM pg_dist_transaction;
+ count 
+-------
+     0
+(1 row)
+
+SELECT COUNT(*) FROM pg_dist_node;
+ count 
+-------
+     2
+(1 row)
+
+SELECT COUNT(*) FROM pg_dist_colocation;
+ count 
+-------
+     0
+(1 row)
+
+SELECT last_value FROM pg_dist_colocationid_seq;
+ last_value 
+------------
+          1
+(1 row)
+
+SELECT last_value FROM pg_dist_groupid_seq;
+ last_value 
+------------
+          2
+(1 row)
+
+SELECT last_value FROM pg_dist_node_nodeid_seq;
+ last_value 
+------------
+          2
+(1 row)
+
+SELECT last_value FROM pg_dist_shard_placement_placementid_seq;
+ last_value 
+------------
+         30
+(1 row)
+
+SELECT last_value FROM pg_dist_shardid_seq;
+ last_value 
+------------
+    1360000
+(1 row)
+
+SELECT last_value FROM pg_dist_jobid_seq;
+ last_value 
+------------
+    1360000
+(1 row)
+
+RESET role;
+DROP USER no_access;
```

Comment:
Hm. This seems very likely to get out of date. I think we should rather do something like:
```
-- list relations in the citus extension without sufficient privileges
SELECT pg_class.oid::regclass
FROM pg_class
     JOIN pg_namespace nsp ON (pg_class.relnamespace = nsp.oid)
     JOIN pg_depend dep ON(objid = pg_class.oid)
     JOIN pg_extension ext ON (ext.oid = dep.refobjid)
WHERE 
    refclassid = 'pg_extension'::regclass
    AND classid ='pg_class'::regclass
    AND ext.extname = 'citus'
    AND nsp.nspname = 'pg_catalog' 
    AND NOT has_table_privilege(pg_class.oid, 'select')
;
```